packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -284,7 +284,7 @@ export const EditorFormComponent = ({form, formType, formProps, document, name,
       });
       const cleanupSuccessForm = context.addToSuccessForm((result: any, form: any, submitOptions: any) => {
         getLocalStorageHandlers(currentEditorType).reset();
-        if (editorRef.current && !submitOptions?.redirectToEditor) {
+        if (editorRef.current && !submitOptions?.redirectToEditor && !isCollabEditor) {
```

Comment:
What is this change intended to do/prevent/etc?

Reply:
If you don't make this change, then we try to call `setData` during a collaborative editing session, which crashes the page.